doc/get-started/quick-start.rst

```diff
@@ -0,0 +1,59 @@
+Quick start
+-----------
+We'll cover:
+* Installation
+* Get the ML-basic example
+* Run Ploomber
+
+Installation
+------------
+
+Using ``pip``
+*************
+
+.. code-block:: console
+
+    pip install ploomber
+
+
+Using ``conda``
+***************
+
+.. code-block:: console
+
+    conda install ploomber -c conda-forge
+
+
+
+Get an example
+--------------
+
+.. code-block:: console
+    # ML pipeline example
```

Comment:
this wont render well, it needs an empty line, the other code blocks  need it as well